package/yast2-network.changes

```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Fri Feb 12 17:31:48 UTC 2016 - knut.anderssen@suse.com
+
+- bsc#903190
+  - Recognized netcards with subclass 02 as InfiniBand (ib) which
```

Comment:
1) AFAIK it should be 06
2) I'd go for more "user's" friendly description here. Something like "Improved detection of InfiniBand cards".